MetabolicAtlas/MetabolicAtlas
# feat: british english spell check

**Related issue(s) and PR(s)**  
This PR closes #1017 and relates to https://github.com/MetabolicAtlas/data-files/pull/30.



**Type of change**  

 
**List of changes made**  

- Add cspell to frontend and api
- Fix typos and american spellings

**Screenshot of the fix**  


**Testing**  

- Make sure the web page is functioning as before and that all the changes look good
- Make sure the file `project-words.txt` does not contain any faulty words
- Make sure the github actions are looking good

**Further comments**  

- Note that we are not spell checking files under `api/src/data` as they are not owned by us and we should not enforce British spelling onto the researchers. I have created https://github.com/MetabolicAtlas/data-files/pull/30 for a one time fix of `gemRepository.json` typos
- If you want to see how it works when the spelling fixes are not applied, check out 64c0a593ce20d25be376ba59266134bfba1fab05. Either try pushing something to github (if so remove it afterwards) or run `yarn cspell '**/src/*/*.{js,vue,txt,md,json}'` locally in folder `api` or `frontend`

**Definition of Done checklist**
